README.md

```diff
@@ -1,25 +1,118 @@
-# Sivy
+# Sivy  &middot; [![NPM version](https://img.shields.io/npm/v/@indec/sivy.svg)](https://www.npmjs.com/package/@indec/sivy) [![Build Status](https://travis-ci.org/indec-it/sivy.svg?branch=master)](https://travis-ci.org/indec-it/sivy) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/indec-it/sivy/blob/master/LICENSE)
 
 The survey sincronization server for MongoDB.
 
-## Installation
+## How to use
 
-    npm install @indec/sivy
+### Setup
 
-## Environment variables
+```bash
+npm install @indec/sivy
```

Comment:
Add `--save`

Reply:
node@8 comes with npm@5, the `--save` is on by default.